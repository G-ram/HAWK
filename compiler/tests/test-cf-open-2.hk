BEGIN{}
END{
    open("a");
    print("open 2 successful")
}
