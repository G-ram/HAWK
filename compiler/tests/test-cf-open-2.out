open 2 successful
